styles/style.css

```diff
@@ -0,0 +1,312 @@
+@import url('https://use.fontawesome.com/releases/v5.8.1/css/all.css');
+
+body {
+  font-family: 'IBM Plex Sans', sans-serif;
+  font-size: 16px;
+  color: #676a81;
+  background: rgb(225, 247, 242);
+  max-width:1920px;
+  margin:0 auto;
+  overflow-x:hidden;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  margin: 0;
+  color:#43485c;
+  font-size: 20px;
+  font-family: 'IBM Plex Sans', sans-serif;
+  text-transform: capitalize;
+}
+
+p {
+  margin: 0;
+  line-height: 1.8;
+  color: #43485c;
+  font-size: 14px;
+  font-family: 'IBM Plex Sans', sans-serif;
+}
+
+img {
+  border: none;
+  max-width: 100%;
+  height: auto;
+}
+
+ul {
+  padding: 0;
+  margin: 0 auto;
+  list-style: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+[placeholder]:focus::-webkit-input-placeholder {
+  -webkit-transition: opacity 0.3s 0.3s ease;
+  -moz-transition: opacity 0.3s 0.3s ease;
+  -ms-transition: opacity 0.3s 0.3s ease;
+  -o-transition: opacity 0.3s 0.3s ease;
+  transition: opacity 0.3s 0.3s ease;
+  opacity: 0;
+}
+
+a, a:hover, a:active, a:focus {
+  display: inline-block;
+  text-decoration: none;
+  color: #6a708e;
+  font-size: 16px;
+  padding: 0;
+}
+
+nav.navbar {
+  background-color: #fff;
+  border-bottom: solid 1px #e0e0e0;
+  box-shadow: 0 3px 15px rgba(0,0,0,.2);
+  font-family: 'IBM Plex Sans', 'Open Sans', sans-serif;
+  font-size: 18px;
+  top: 5px;
+  position: fixed;
+  align-items: center;
+  display: block;
+  width: 100%;
+  height: 50px;
+  z-index: 9;
+}
+
+nav.navbar ul.nav {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+
+nav.navbar ul.nav > li > a{
+  color: #6f6f6f;
+  padding: 15px 20px;
+  background-color: transparent;
+  outline: none;
+  margin-bottom: -2px;
+  font-size: 16px;
+  font-weight: 400;
+  text-transform: uppercase;
+  transition: 0.3s linear;
+}
+
+.nav li:not(:first-child)not(:last-child){
+  margin-right: 10px;
+}
+
+nav.navbar > ul.nav > li > a:hover,
+nav.navbar > ul.nav > li > a:focus,
+nav.navbar > ul.nav > li.active > a {
+  color: #d92cf9;
+}
+
+nav.navbar > ul.nav > li.navbar-name a,
+nav.navbar > ul.nav > li.navbar-name a:hover,
+nav.navbar > ul.nav > li.navbar-name a:focus > li > a {
+  display: inline-block;
+  color: #d92cf9;
+  margin-left: 12px;
+  font-size: 20px;
+  font-weight: 600;
+  padding: 15px 33px 0px 20px;
+  text-transform: uppercase;
+  height: 50px;
+  line-height: 20px;
+}
+
+.welcome {
+  display: flex;
+  flex-flow: column wrap;
+  text-align: center;
+  font-family: 'Open Sans', sans-serif;
+  margin: 180px 0px 0px 0px;
+  padding: 5px 0px 15em 0px;
+}
+
+header > h1{
+  color: #fff;
+  font-size: 54px;
+  font-weight: 700;
+  text-transform: uppercase;
+  line-height: 1.5;
+}
+
+.welcome sub {
+  font-size: 1.5em;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: bold;
+  text-transform: capitalize;
+  font-stretch: ultra-expanded;
+  margin-bottom: 2em;
+  width: 100%;
+}
+
+.welcome a[href] {
```

Comment:
Does this selector do anything other than select the `a` element? All the `a` elements have `href` attributes.